dags/public-health/covid19/shelter-to-esri.py

```diff
@@ -67,14 +67,20 @@ def load_data(**kwargs):
     
     # timestamp 
     df['Timestamp'] = pd.to_datetime(df.Timestamp).dt.tz_localize(tz="US/Pacific").dt.tz_convert('UTC')
+    df['string-ts'] = df['Timestamp'].dt.strftime('%B %d, %Y, %r')
+
+    df['string-ti'] = df['Time'].dt.strftime('%B %d, %Y, %r')
+    # do something horrifying 
+    df['string-ts'].iloc[1] = "None"
```

Comment:
Wow, this really worked?

Reply:
unfortunately, yes?

Reply:
![image](https://user-images.githubusercontent.com/1735650/77786630-5cb4c480-701b-11ea-93b5-497d1332a9ab.png)

Reply:
Debugging AGOL is like navigating by brownian motion.